sdk/python/kfp/v2/v2_namespace_alias_test.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2020 The Kubeflow Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# pylint: disable=import-outside-toplevel,unused-import,import-error
+import unittest
+
+
+class V2NamespaceAliasTest(unittest.TestCase):
+    """Test that imports of both modules and objects are aliased
+    (e.g. all import path variants work).
+    """
+
+    def test_1_import_module(self):  # pylint: disable=no-self-use
+        from kfp.v2 import dsl as dsl2
```

Comment:
Maybe you can add the check warnings here, check whether warnings are right. 

And for usage, we normally use:
```
from kfp.v2 import dsl

@dsl.pipeline(name="xxx")
def pipeline:
etc.
``` 
Try to put a real usage here, as we sometimes refer to these usages from tests.

Reply:
Good idea on adding usage. Done.

I worked on checking that the warning is raised using `self.assertWarns` but it turned out to be fairly challenging to implement reliably and probably not worth the effort, I think. The issue is that `unittest` appears to use the same execution session for all tests, so it's not certain that the `kfp.v2` path will be hit for the first time in the  `sdk/python/kfp/v2/v2_namespace_alias_test.py` file (in fact, this has never happened for me in practice). Once the `kfp.v2` module is loaded (by another unrelated test, in this case), the `DeprecationWarning` is not raised. I tried a few workarounds using `sys.modules`, but this was unsuccessful.